src/core/lib/promise/activity.h

```diff
@@ -170,8 +178,8 @@ class Activity : private Wakeable {
   static bool have_current() { return g_current_activity_ != nullptr; }
   // Check if we got an internal wakeup since the last time this function was
   // called.
-  bool got_wakeup() ABSL_EXCLUSIVE_LOCKS_REQUIRED(mu_) {
-    return absl::exchange(got_wakeup_during_run_, false);
+  ActionDuringRun got_action_during_run() ABSL_EXCLUSIVE_LOCKS_REQUIRED(mu_) {
```

Comment:
nit: since this is not a plain accessor, function names should be mixed case, like `GotActionDuringRun()`. See https://google.github.io/styleguide/cppguide.html#Function_Names. This class has a few such issues, so it can be handled in another cleanup (or argued, it's debatable).

Reply:
done